flow/connectors/sql/query_executor.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/PeerDB-io/peer-flow/model/qvalue"
 )
 
+//nolint:iface
 type SQLQueryExecutor interface {
```

Comment:
Long term we should just get rid of this, only used by e2e, only implemented with snowflake/sqlserver